test/taskcreator_test.js

```diff
@@ -10,10 +10,12 @@ suite('TaskCreator', function() {
   this.slow(500);
   helper.setup();
 
-  // these tests require real TaskCluster credentials (for the queue insert)
-  if (!helper.haveRealCredentials && !process.env.TASK_ID) {
-    this.pending = true;
-  }
+  setup(function() {
```

Comment:
Putting this in a `setup` block makes it run after helper's `setupSuite` has run (which is what sets `helper.haveRealCredentials`). Without this, these tests never run! /cc @catlee fiennyangeln